### ISSUE TITLE
ERROR No sigchain was available

### ISSUE THREAD

**Issue by mike-pittelko** [keybase/keybase-issues#2541]
When attempting to follow or track someone, I keep getting the error "ERROR No sigchain was available" - which isn't very helpful.  It's totally unclear if I'm doing something wrong (I don't think so - keybase track foo_user) or if something is wrong in keybase.  I'm positive foo_user exists, and a search results in plenty of identity information.

**Comment by maxtaco** [comment]
Which command gave this error message?

**Comment by mike-pittelko** [comment]
keybase track foo_user 
or
keybase follow foo_user

I'm positive foo_user exists, and a search results in plenty of identity information.

**Comment by pfasante** [comment]
I have the same error. Additionally, when searching for foo_user I get the following warning:

> ▶ WARNING Run error: RPC error. type: 0, method: keybase.1.apiserver.Get, length: 4, error: error decoding message field at position 3, error: [pos 29]: readContainerLen: Unrecognized descriptor byte: hex: 83, decimal: 131

**Comment by maxtaco** [comment]
Which platform?

**Comment by pfasante** [comment]
I'm running an (X)ubuntu 16.04 with kernel 4.4.0

**Comment by maxtaco** [comment]
@pfasante that second error is quite worrisome.  Can you please issue a `keybase log send` and we'll look into it?  Many thanks

CC: @jzila

**Comment by maxtaco** [comment]
Also, can you guys be slightly more specific -- which user are you referring to?  Or does this happen for all users?

**Comment by pawelad** [comment]
Same here, OS X 10.11.6, Keybase 1.0.17 (from homebrew):

```
→ keybase id foo
▶ INFO Stopping keybase.service
▶ INFO Stopping homebrew.mxcl.keybase
▶ INFO Saving /Users/pawelad/Library/LaunchAgents/homebrew.mxcl.keybase.plist
▶ INFO Starting homebrew.mxcl.keybase
▶ ERROR No sigchain was available

→ keybase id foo
▶ ERROR No sigchain was available

→ keybase follow foo
▶ ERROR No sigchain was available
```

First time trying to use the application : -)

**Comment by maxtaco** [comment]
@pawelad how did you install keybase?

CC: @oconnor663 on the `no sigchain was available error`, which is likely the wrong error.

**Comment by maxtaco** [comment]
Also, @pawelad, can you issue a `keybase log send`

Everyone on this thread, these are likely configuration/installation issues that we need to get to the bottom of.

**Comment by pawelad** [comment]
@maxtaco I installed it via homebrew: `$ brew install keybase`.

Log ID: 46217a5b146de548959e331c

**Comment by pawelad** [comment]
@maxtaco `keybase id max` on the other hand works as expected. I looked into the the profile I wanted to follow (that I just invited) and he has no PGP keys - I'm guessing that's the reason.

I probably shouldn't be able to ID him(?), but a nice error message would be nice : -)

**Comment by maxtaco** [comment]
Yeah, we agree. Just trying to get to the bottom of what's actually going here.  @pawelad in your case the install worked, but you weren't able to ID that user, because his account is basically empty.

**Comment by mike-pittelko** [comment]
My install was done using the downloaded installer.

**Comment by pfasante** [comment]
I tried a "keybase follow phl" -- also a user I invited which has no other keys or so connected to its keybase account.

@maxtaco I submitted the keybase log, its ID is: 6883757ce08b94c4a46de71c. During the operation I got the following warning:
> ▶ WARNING error opening log "~/.cache/keybase/keybase.updater.log": open ~/.cache/keybase/keybase.updater.log: no such file or directory
(I issued the errenous 'keybase follow phl', before sending the log)
